.github/workflows/main.yml

```diff
@@ -25,6 +21,7 @@ jobs:
         go test -vet=off -timeout=15m -race -covermode atomic -coverprofile=covprofile ./...
 
     - name: Go analyze
+      shell: bash
```

Comment:
can we just make bash the default shell for the entire job?

Reply:
shell is a property of run, not of the job, as per https://json.schemastore.org/github-workflow

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_iddefaultsrun

Reply:
Great, I will use this default instead then